svc-managers/managers/managers.go

```diff
@@ -313,10 +313,29 @@ func (e *ExternalInterface) VirtualMediaActions(req *managersproto.ManagerReques
 	}
 	// splitting managerID to get uuid
 	requestData := strings.Split(req.ManagerID, ":")
-	log.Info(requestData)
 	uuid := requestData[0]
-	log.Info(req.URL, uuid, string(requestBody))
 	resp = e.deviceCommunication(req.URL, uuid, requestData[1], http.MethodPost, requestBody)
+
+	// If the virtualmedia action is success then updating DB
+	if resp.StatusCode == http.StatusOK {
```

Comment:
Please handle error scenarios too.

Reply:
This is only for updating DB, there is nothing to update in DB if the operation is failed